src/Console/ConfigurationResolver.php

```diff
@@ -832,10 +832,10 @@ private function resolveFinder(): iterable
 
         $isIntersectionPathMode = self::PATH_MODE_INTERSECTION === $this->options['path-mode'];
 
-        $paths = array_filter(array_map(
+        $paths = array_map(
             static fn (string $path) => realpath($path),
             $this->getPath()
-        ));
+        );
```

Comment:
`array_filter()` was here probably to cover possible `false` returned from `realpath()`. I am not sure when this can happen though, because if you provide non-existing path in CLI command, `getPath()` throws an exception:

```
In ConfigurationResolver.php line 382:
                                                                          
  [PhpCsFixer\ConfigurationException\InvalidConfigurationException (16)]  
  The path "src/Config2.php" is not readable.
```

Anyway, without `array_filter()` it's possible that `$paths` contain `false`. @kubawerlos are you sure we won't hit this?

Reply:
I wasn't able to add a test for it - method `ConfigurationResolver::getPath` seems to take care of that pretty well by throwing an exception for incorrect path.